src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -28,33 +32,102 @@ Intermediary is an edn (https://github.com/edn-format/edn) data format
 that allows defining data rows for marketplaces. It supports creating
 links between rows via references.
 
+We recommend that you use an .edn library to generate the data and
```

Comment:
Maybe we could provide a link to a library or two that could be used to generate and validate the syntax? Or are we aware of any we can recommend?

Reply:
That's a good idea! If @lyyder has ideas on what to recommend that would be great 👍